tests/moses/scoring/discretize_contin_bscoreUTest.cxxtest

```diff
@@ -0,0 +1,70 @@
+/** discretize_contin_bscore_UTest.cxxtest ---
+ *
+ * Copyright (C) 2019 OpenCog Foundation
+ *
+ * Authors: Kasim Ebrahim <se.kasim.ebrahim@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License v3 as
+ * published by the Free Software Foundation and including the exceptions
+ * at http://opencog.org/wiki/Licenses
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program; if not, write to:
+ * Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+
+#include <cxxtest/TestSuite.h>
+#include <opencog/moses/scoring/bscores.h>
+#include <opencog/atoms/base/Link.h>
+#include <opencog/data/table/table.h>
+#include <opencog/data/table/table_io.h>
+
+#include "opencog/utils/value_key.h"
+
+using namespace opencog;
+using namespace combo;
+using namespace moses;
+using namespace std;
+
+const string utest_dir=string(PROJECT_SOURCE_DIR) + "/tests/moses/scoring/";
+const string target_feature="out";
+const vector<double> thresholds={0};
+
+class discretize_contin_bscoreUTest : public CxxTest::TestSuite
+{
+public:
+	void test_disc_contin_bscore(void)
+	{
+		Table table = loadTable(utest_dir + "disc_contin_data.csv", target_feature);
+		discretize_contin_bscore dis_c_bscore(table.otable, table.itable, thresholds, false);
+
+		Handle f1 = createNode(SCHEMA_NODE, "f1");
+		Handle f2 = createNode(SCHEMA_NODE, "f2");
+		std::vector<double > _f1_values;
+		std::vector<double > _f2_values;
+
+		boost::transform(table.itable.get_column_data(0), back_inserter(_f1_values), get_contin);
+		boost::transform(table.itable.get_column_data(1), back_inserter(_f2_values), get_contin);
+
+		ValuePtr f1_proto_atom(new FloatValue(_f1_values));
+		f1->setValue(moses::value_key, f1_proto_atom);
+		ValuePtr f2_proto_atom(new FloatValue(_f2_values));
+		f2->setValue(moses::value_key, f2_proto_atom);
+
+		HandleSeq handleSeq = {f1, f2};
+		Handle program = createLink(handleSeq, PLUS_LINK);
```

Comment:
You could directly write
```suggestion
		Handle program = createLink({f1, f2}, PLUS_LINK);
```
It's fine as it is, just saying in case you didn't know.